pandas/io/formats/style.py

```diff
@@ -547,13 +554,33 @@ def to_latex(
         >>> df = pd.DataFrame([[1,2], [3,4]])
         >>> s = df.style.highlight_max(axis=None,
         ...                            props='background-color:red; font-weight:bold;')
-        >>> s.render()  # doctest: +SKIP
+        >>> s.render()  # doctest: +ELLIPSIS
+        '<style type="text/css">\n#T_..._row1_col1 {\n  background-color: red;\n
+        font-weight: bold;\n}\n</style>\n<table id="T_..._">\n  <thead>\n
+        <tr>\n      <th class="blank level0" >&nbsp;</th>\n
+        <th class="col_heading level0 col0" >0</th>\n
+        <th class="col_heading level0 col1" >1</th>\n
+        </tr>\n  </thead>\n  <tbody>\n    <tr>\n
+        <th id="T_..._level0_row0" class="row_heading level0 row0" >0</th>\n
+        <td id="T_..._row0_col0" class="data row0 col0" >1</td>\n
+        <td id="T_..._row0_col1" class="data row0 col1" >2</td>\n    </tr>\n
+        <tr>\n      <th id="T_..._level0_row1" class="row_heading level0 row1" >1</th>\n
+        <td id="T_..._row1_col0" class="data row1 col0" >3</td>\n
+        <td id="T_..._row1_col1" class="data row1 col1" >4</td>\n    </tr>\n
+        </tbody>\n</table>\n...'
 
         The equivalent using LaTeX only commands is the following:
 
         >>> s = df.style.highlight_max(axis=None,
         ...                            props='cellcolor:{red}; bfseries: ;')
-        >>> s.to_latex()
+        >>> s.to_latex()  # doctest: +SKIP
+        \begin{tabular}{lrr}
+        {} & {0} & {1} \\
+        0 & 1 & 2 \\
+        1 & 3 & \\
+        cellcolor{red} \\
+        bfseries 4 \\
+        \end{tabular}
```

Comment:
similarly here, I don't think we need to show the latex. The point of this section in the docstring is to show how to create HTML and the corresponding LaTeX outputs. The sections that succeed this introduction display the results, in terms of LaTeX outputs.